org.eclipse.scanning.event/src/org/eclipse/scanning/event/ConsumerImpl.java

```diff
@@ -496,6 +519,28 @@ public void run() throws EventException {
 		}
 	}
 	
+	private void checkStartPaused() throws EventException {
+		
+		if (!isPauseOnStart()) { 
+			return;
+		}
+		
+		List<U> items = getSubmissionQueue();
+		if (items!=null && items.size()>0) {
+			pause();
+
+			IPublisher<PauseBean> pauser = eservice.createPublisher(getUri(), IEventService.CMD_TOPIC);
+			pauser.setStatusSetName(IEventService.CMD_SET); // The set that other clients may check
+			pauser.setStatusSetAddRequired(true);
+
+			PauseBean pbean = new PauseBean();
+			pbean.setQueueName(getSubmitQueueName()); // The queue we are pausing
+			pbean.setPause(true);
+			pauser.broadcast(pbean);
+
+		}
+	}
+
 	private void checkPaused() throws Exception {
 		
 		if (!isActive()) throw new Exception("The consumer is not active and cannot be paused!");
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Define and throw a dedicated exception instead of using a generic one. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS00112)
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') At most one statement is allowed per line, but 2 statements were found on this line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS00122)